component/componenttest/nop_receiver_test.go

```diff
@@ -33,17 +33,17 @@ func TestNewNopReceiverFactory(t *testing.T) {
 	cfg := factory.CreateDefaultConfig()
 	assert.Equal(t, &nopReceiverConfig{ReceiverSettings: config.NewReceiverSettings(config.NewID("nop"))}, cfg)
 
-	traces, err := factory.CreateTracesReceiver(context.Background(), component.ReceiverCreateParams{}, cfg, consumertest.NewNop())
+	traces, err := factory.CreateTracesReceiver(context.Background(), component.ReceiverCreateSettings{}, cfg, consumertest.NewNop())
```

Comment:
Just as the other test, it may be useful to have a nop instance. Not a biggie though.

Reply:
Sure, but I didn't want to change any logic there. Only refactor the code